pkg/bff/gds.go

```diff
@@ -24,8 +24,10 @@ import (
 )
 
 const (
-	testnet = "testnet"
-	mainnet = "mainnet"
+	testnet       = "testnet"
+	mainnet       = "mainnet"
+	trisatest     = "trisatest.net"
+	vaspdirectory = "vaspdirectory.net"
```

Comment:
Just confirming; are we figuring that internally to the bff we refer to the test network as `testnet` but that users of the bff will reference it as `trisatest.net`? Just want to make sure we document our reason for referencing the two networks in multiple ways so that we don't get mixed up later!

Reply:
This is sort of a weird situation: the GDS thinks of itself as `trisatest.net` or `vaspdirectory.net` - so when the GDS emails users, that's how it will represent itself. This makes sense from the perspective of the GDS since there is the possibility of multiple directory services that are respected by TRISA. Users (e.g. VASPs/TRISA implementers) should also think about the directories as `trisatest.net` and `vaspdirectory.net` since those are the endpoints they will be connecting to (though there is a lot of use of the colloquial "MainNet" and "TestNet" conversationally - but this tends to always require clarification, even now). 

The BFF internally thinks about `testnet` and `mainnet`; but unfortunately BFF users (e.g. the UI) will have to understand both just like the BFF has to understand both (though `testnet` and `mainnet` will be the common case). This is for two reasons:

1. In URLs with path parameters, e.g. `/v1/register/:network` we have to use `testnet` and `mainnet` because `vaspdirectory.net` and `trisatest.net` are not valid. They are valid in query parameters, e.g. `/v1/verify?network=trisatest.net` 
2. In copy the UI will have to make sure it uses "TestNet" and "MainNet" to describe the different networks/certificates but "trisatest.net" and "vaspdirectory.net" as the directories, and recommend to users that they should connect to the directories via "api.trisatest.net:443" and "api.vaspdirectory.net:443" respectively. 

Most of this will be made easier in BFF v1.5 - we plan to have a single verification token which is sent from the BFF - that way the contact is verified for both the TestNet and the MainNet with a single email; that will eliminate case #1 - but not case #2. 

As an alternative we could replace "testnet" with the "trisatest" and "mainnet" with "vaspdirectory", making the .net TLD optional for things like case #1. I'm certainly open to that.